build/pandoc/defaults/html.yaml

```diff
@@ -3,6 +3,9 @@
 to: html5
 output-file: output/manuscript.html
 include-before-body:
```

Comment:
Will need to comment this out as well so it's not interpreted as having a null value.